pkg/dir.traversal.targets

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="..\dir.traversal.targets" />
+  
+  <PropertyGroup Condition="'$(OfficialBuildId)' != ''">
+    <!-- During an official build, only build identity packages in the AllConfigurations build -->
+    <SkipBuildIdentityPackage Condition="'$(BuildAllConfigurations)' != 'true'">true</SkipBuildIdentityPackage>
```

Comment:
This change is in corefx.  Also the logic you're pointing out is pre-existing from an PR last week.

In core-setup there is a flag that indicates the package is signed.  I also think in core-setup we don't need this logic because it already puts packages in leg-specific paths and prefers one leg over others.